### PR TITLE
Renamed npm pckage to `theia-ide` because a theia package already exists on npm.

### PR DESCRIPTION
**Pull request by svenefftinge**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "theia",
+  "name": "theia-ide",
   "version": "0.0.1-alpha.1",
   "description": "Theia - JavaScript IDE Framework",
```
Comment (akosyakov) [inline package.json]: Could we also update the description to the same which we use at github: `Theia is a cloud & desktop IDE implemented in TypeScript.`

**Comment by hexa00** [comment]
That doesn't work for me.

I thought theia was a framework to create IDEs and we thought of calling the acual IDE the theia-ide... Note that could be something else

But I think something like theia-framework something like that would be more clear otherwise it's not clear what is the framework and what is the product based on it ?

File: package.json (hunk 1 above)
Comment (svenefftinge) [inline package.json]: +1. How about
`Theia is a cloud & desktop IDE framework implemented in TypeScript.`?

**Comment by svenefftinge** [comment]
Framework is very generic. Theia is for IDEs, no?
Long term I'd like to go with multiple npm packages `theia-core`, `theia-filesystem`, `theia-languages`, etc. We could use `theia-core` now also and then later extract functionality to separate extension packages.

File: package.json (hunk 1 above)
Comment (akosyakov) [inline package.json]: Sounds good

Comment (kittaakos) [inline package.json]: Does that mean that "we" modify the Github description too?

**Comment by hexa00** [comment]
I like theia-core good idea please go for that

**Pull request merged**